### ISSUE TITLE
Cart content is not read by screen readers

### ISSUE THREAD

**Issue by bschulzebaek** [shopware/platform#9229]
### Shopware Version

6.7.0

### Affected area / extension

Platform(Default)

### Actual behaviour

When a user visits the Storefront and Screen Readers read out the page structure/content, the header cart widget is described as a link, the content (number of items, total amount) is ignored.

### Expected behaviour

Screen readers should be able to describe the current cart content when visiting the page.

### How to reproduce

0. Enable a Screen Reader
1. Visit the Storefront and let it do it's thing describing the page / tab-navigate to the header cart icon
2. VO describes the link, but not the content

**Comment by bschulzebaek** [comment]
Btw: Fixing this should also cover cart updates, since the page is reloaded after changes and the content would be read again. 💡
